service/src/main/java/uk/gov/hmcts/reform/fpl/model/ApplicantParty.java

```diff
@@ -8,15 +8,24 @@
 import uk.gov.hmcts.reform.fpl.model.common.EmailAddress;
 import uk.gov.hmcts.reform.fpl.model.common.Party;
 import uk.gov.hmcts.reform.fpl.model.common.Telephone;
+import uk.gov.hmcts.reform.fpl.validators.interfaces.HasApplicantContactName;
+import uk.gov.hmcts.reform.fpl.validators.interfaces.HasApplicantTelephone;
 
 import java.util.Date;
 
+import javax.validation.Valid;
+import javax.validation.constraints.NotBlank;
+
 @Data
 @EqualsAndHashCode(callSuper = true)
+@HasApplicantTelephone
+@HasApplicantContactName
 @JsonInclude(JsonInclude.Include.NON_NULL)
 public class ApplicantParty extends Party {
+    @NotBlank(message = "Enter the applicant's full name")
     private final String organisationName;
     private final Telephone mobileNumber;
+    @NotBlank(message = "Enter a job title for the contact")
```

Comment:
"Enter the job title of/for the person to contact" to me reads better than enter a job title for the contact , but again it is just preference and if this is what ux and design wants that is fine :)

Reply:
Might be worth raising with Theo Becky. But that's from the AC :)

Reply:
Oh ok if thats from AC thats fine, my following comments are similar then I just think some of them dont read very well but that could just be personal preference.

Reply:
Yeah it's very subjective. Worth bringing up with UX though.